modules/src/joke.py

```diff
@@ -1,13 +1,17 @@
 import re
 import requests
+from random import randint
 
 def process(input, entities=None):
     output = {}
     try:
-        r = requests.get('https://api.chucknorris.io/jokes/random')
-        data = r.json()
         output['input'] = input
-        output['output'] = data['value']
+        if randint(0,1):
+            data = requests.get('https://api.chucknorris.io/jokes/random').json()
+            output['output'] = data['value']
+        else:
+            data = requests.get('http://api.yomomma.info/').json()
+            output['output'] = data['joke']
```

Comment:
Also, some of these jokes are pretty offensive (being yo momma jokes), I wonder if it's appropriate to have it on a chat bot... (We wouldn't want a [Tay ](https://en.wikipedia.org/wiki/Tay_(bot))fiasco :) )